src/vm/eventpipebuffer.cpp

```diff
@@ -70,19 +69,25 @@ bool EventPipeBuffer::WriteEvent(Thread *pThread, EventPipeSession &session, Eve
 
     // Calculate the location of the data payload.
     BYTE *pDataDest = m_pCurrent + sizeof(EventPipeEventInstance);
+    EventPipeEventInstance *pInstance;
 
     bool success = true;
     EX_TRY
     {
+        LPCGUID pThreadActivityID = pActivityId;
```

Comment:
Is this assignment intended?

Reply:
Yeah I missed that. Thanks for catching that